### PR TITLE
Update ligand_networks_for_developers

### PR DESCRIPTION
**Pull request by dwhswenson**
- [x] Removed a bunch of debug stuff (which caused problems on GitHub for some browsers).
- [x] Added something about writing ligand networks as GraphML.

The reason to write to GraphML is that right now, that's the best way to use the visualization tool (via the horrible incantation `python -m openfe.utils.atommapping_network_plotting network.graphml` -- we should probably put that somewhere more easily accessible.)

### CODE REVIEW

File: networks/ligand_networks_for_developers.ipynb [hunk 1]
```diff
@@ -1678,13 +955,26 @@
...
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "fe507fdb",
+   "metadata": {},
+   "source": [
+    "## Writing a ligand network to disk\n",
+    "\n",
+    "If you want to save a ligand network to disk in order to transfer it elsewhere, you can do some with the `to_graphml()` method."
```
Comment (IAlibay) [inline networks/ligand_networks_for_developers.ipynb:965]:
```suggestion
    "If you want to save a ligand network to disk in order to transfer it elsewhere, you can do so with the `to_graphml()` method."
```

**Review by IAlibay — APPROVED** [review]
Just the one typo, I'll approve eagerly.

**Comment by dwhswenson** [comment]
Ran this with latest releases, then tested again in my env with both gufe and openfe dev versions.

**Comment by richardjgowers** [comment]
I've put these changes into #42

**Pull request closed without merging**